### PR TITLE
Fixing a bug with multi-cuke that made it ignore scenario outlines an…

### LINKED ISSUE #46: Support scenario outlines
multi-cuke doesn't seem to correctly support outlines. If I have this scenario:

```
Feature: LKG
  @LKG
  Scenario Outline: No-op test
    Given I do nothing

    Examples:
      | iteration |
      |     1     |
      |     2     |
      |     3     |
```

When I run it in regular cucumber I get:

```
Feature: LKG

  @LKG
  Scenario: No-op test
    Given I do nothing

  @LKG
  Scenario: No-op test
    Given I do nothing

  @LKG
  Scenario: No-op test
    Given I do nothing

3 scenarios (3 passed)
3 steps (3 passed)
```

But when I run it in multi-cuke I get:

```
Feature: LKG
  @LKG
  Scenario: No-op test
    Given I do nothing  # lkg.feature:4

1 scenario (1 passed)
5 steps (5 passed)
```

### PR DESCRIPTION
**Pull request by vsiakka**
…d pinning the cucumber version to 1.0.0 instead of ^1.0.0 because the latest version of cucumber breaks the parser

@guykisel @midniteio 

This fixes https://github.com/midniteio/multi-cuke/issues/46.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "bluebird": "^3.3.1",
     "commander": "^2.9.0",
-    "cucumber": "^1.0.0",
+    "cucumber": "1.0.0",
```
Comment (guykisel) [inline package.json:29]: why this change?

Comment (guykisel) [inline package.json:29]: edit: nevermind, read the description. should we also fix our compatibility with the latest cucumber?

File: src/lib/feature-finder.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ export default function(cucumberOptions) {
       let featureData = parseFeature(file);
       return featureData.children
         .filter((child) => {
-          return child.type === "Scenario" && verifyTags(child, cucumberOptions.tags);
+          return (child.type === "Scenario" || child.type === "ScenarioOutline") && verifyTags(child, cucumberOptions.tags);
```
Comment (guykisel) [inline src/lib/feature-finder.js]: glad to see this is so simple!

**Comment by guykisel** [comment]
looks like this failed lint. otherwise :+1:

File: package.json (hunk 1 above)
Comment (vsiakka) [reply package.json:29]: We should, and we should bump the major version of this repo because mutil-cuke allows you to specify the cucumber you want to use and the newer version of cucumber creates a backwards incompatible output.
